app/controllers/items/new.js

```diff
@@ -463,13 +504,29 @@ export default GoodcityController.extend(
             this.printBarcode(data.item.id);
           }
           this.updateStoreAndSaveImage(data);
-          this.clearSubformAttributes();
-          this.setProperties({
-            locationId: "",
-            inventoryNumber: "",
-            offersLists: []
-          });
-          this.replaceRoute("items.detail", data.item.id);
+          if (this.get("isDuplicate") && !this.get("isBoxOrPallet")) {
```

Comment:
Not sure about the `isDuplicate` name, doesn't `shouldDuplicate` make more sense ? As in, when we check the box, we indicate that we _want_ to create a duplicate. Not that this item _is_ a duplicate of anything 🤔

Reply:
sure will change that